descriptors/ModuleDescriptor-template.json

```diff
@@ -667,6 +674,11 @@
       "displayName": "Invoice Line - get collection of Invoice lines",
       "description": "Get collection of Invoice lines"
     },
+    {
+      "permissionName": "invoice.invoice-lines.fund-distributions.validate",
+      "displayName": "Orders - validate fund distributions for a invoice line",
```

Comment:
This should be Invoices

Reply:
done.

Reply:
Maybe I wrote suggestion not using much details, we need to rename this line "Orders - validate fund distributions for a invoice line" to this line "Invoices - validate fund distributions for a invoice line".
We don't need to touch permission names, if all permissions named in singular manner - we need to use the same pattern for new permission as well